ReactTask/src/App.js

```diff
@@ -3,19 +3,37 @@ import { Switch, BrowserRouter } from 'react-router-dom';
 import { ThemeProvider } from 'styled-components';
 import { Loader } from './components/';
 import { Router } from './Router';
+import { AlertMessage, ResolutionPatient } from './components';
 import theme from './theme';
+import { Provider } from 'react-redux';
+import { store } from './redux/store';
 
```

Comment:
In Router.js, line 34 you have undefined route called doctorProfile. Check this

Reply:
Fixed